docker/cpu.Dockerfile

```diff
@@ -1,3 +1,5 @@
+# DANGER -- If you update this file, make sure to also update gpu.Dockerfile!
```

Comment:
The backend team bases our gpu image on the cpu image so that we usually don't have to update both